orchestra/utils/task_lifecycle.py

```diff
@@ -275,6 +228,182 @@ def assign_task(worker_id, task_id):
     return task
 
 
+@transaction.atomic
+def reassign_assignment(worker_id, assignment_id):
+    worker = Worker.objects.get(id=worker_id)
+    assignment = TaskAssignment.objects.get(id=assignment_id)
+    if assignment.assignment_counter > 0:
+        role = WorkerCertification.Role.REVIEWER
+    else:
+        role = WorkerCertification.Role.ENTRY_LEVEL
+
+    if not _worker_certified_for_task(worker, assignment.task, role):
+        raise WorkerCertificationError('Worker not certified for this task.')
+    if is_worker_assigned_to_task(worker, assignment.task):
+        raise TaskAssignmentError('Worker already assigned to this task.')
+
+    assignment.worker = worker
+    assignment.save()
+    add_worker_to_project_team(worker, assignment.task.project)
+
+    return assignment
+
+
+@transaction.atomic
+def complete_and_skip_task(task_id):
+    task = Task.objects.get(id=task_id)
+    task.status = Task.Status.COMPLETE
+    task.save()
+    for assignment in task.assignments.all():
+        assignment.status = TaskAssignment.Status.SUBMITTED
+        assignment.save()
+    create_subsequent_tasks(task.project)
+    return task
+
+
+@transaction.atomic
+def revert_task_to_datetime(task_id, revert_datetime, fake=False):
```

Comment:
note: I don't love the idea that we're reverting to a datetime rather than reverting to a particular assignment/iteration, since time is a super fuzzy and inaccurate thing in running systems.  We've got assignments/iteration counters to provide a semantic ordering over the iterations, and ideally we'd use that.

But I also realize that implementing `revert_task_to_iteration` might be harder to implement, and if it's significantly harder, I'm not suggesting you do it :).